x/ibc/core/24-host/keys.go

```diff
@@ -64,28 +65,17 @@ func ClientStatePath() string {
 	return "clientState"
 }
 
-// ClientTypePath takes an Identifier and returns Path under which to store the
-// type of a particular client.
-func ClientTypePath() string {
-	return "clientType"
-}
-
 // ConsensusStatePath takes an Identifier and returns a Path under which to
 // store the consensus state of a client.
 func ConsensusStatePath(height exported.Height) string {
-	return fmt.Sprintf("consensusState/%s", height)
+	return fmt.Sprintf("%s/%s", KeyConsensusStatesPrefix, height)
```

Comment:
There is a bug here, and surprised it wasn't flagged by the linters.

Reply:
not sure I follow. What is the bug?